### DISCUSSION TITLE
Listener and target select multiple

### DISCUSSION THREAD

**Discussion by edmaSB** [orchidsoftware/platform#2836] [Q&A] [answered]
hi, I have a listener with a multiple select but the asynchronous call for changing values ​​does not start.

how can I do?

`class DashboardListener extends Listener
{
    
    /**
     * List of field names for which values will be listened.
     *
     * @var string[]
     */
    protected $targets = [
        'mese',
        'prova',
        'cliente_id',

        //'passaggio.tempo_totale',
        'data',
        'dataset',

    ];

    /**
     * The screen's layout elements.
     *
     * @return \Orchid\Screen\Layout[]|string[]
     */
    protected function layouts(): iterable
    {
        return [
            Layout::split([
                Layout::rows(
                    [
                        Input::make('prova')->type('text'),
                        Select::make('mese')
                            ->title(__('Mese'))
                            ->multiple()
                            ->options((new Passaggio())->arrMese)
                            ->autoComplete()
                            //->name('mese-filtro')
                            ->empty('Seleziona il mese'),
                        Select::make('cliente_id')
                            ->title(__('Cliente'))
                            ->multiple()
                            //->name('cliente-filtro')
                            ->fromQuery(Cliente::orderBy('denominazione', 'asc'), 'denominazione')
                            ->autoComplete()
                            ->empty('Seleziona il cliente'),
                        Select::make('data')
                            ->title(__('Data'))
                            ->multiple()
                            //->name('data-filtro')
                            ->fromQuery(Passaggio::select(DB::raw("DATE_FORMAT(data,'%d/%m/%Y') as data"))->distinct('data')->orderBy('data', 'asc'), 'data')
                            ->autoComplete()
                            ->empty('Seleziona la data'),
                    ],
                ),
                DashboardChartLayout::make('dataset'),
            ])->ratio('30/70')->reverseOnPhone(),

        ];
    }

    /**
     * Update state
     *
     * @param \Orchid\Screen\Repository $repository
     * @param \Illuminate\Http\Request  $request
     *
     * @return \Orchid\Screen\Repository
     */
    public function handle(Repository $repository, Request $request): Repository
    {

        //dd($repository); 
        dd($request->input('cliente_id') );
        

        $repository
            ->set('dataset', [
                [
                    'name'   => 'Some Data',
                    'values' => [25, 40, 30, 35, 8, 52, 17],
                    'labels' => ['12am-3am', '3am-6am', '6am-9am', '9am-12pm', '12pm-3pm', '3pm-6pm', '6pm-9pm'],
                ],
                [
                    'name'   => 'Another Set',
                    'values' => [25, 50, -10, 15, 18, 32, 27],
                    'labels' => ['12am-3am', '3am-6am', '6am-9am', '9am-12pm', '12pm-3pm', '3pm-6pm', '6pm-9pm'],
                ],
            ]);

       

        //dd($repository);
        return $repository;
    }
`

**Comment by tabuna** [comment, marked as answer]
In HTML, multiple fields can have names with "[]". For example, if a field was named 'cliente_id', it would be written as `<input name="cliente_id[]">`. To connect the target 🎯, you need to add a dot at the end. Example code:

```php
protected $targets = [
    'client_id.',
];
```
